### ISSUE TITLE
What is the license that applies to this project?

### ISSUE THREAD

**Issue by mz2** [davidcarlisle/web-xslt#4]
What license applies to the XSL source code in this project? It would be great for a license file to be included in the repo.

**Comment by mz2** [comment]
The original Google Code project seems to indicate the MIT license?

**Comment by davidcarlisle** [comment]
On 7 July 2015 at 14:46, Matias Piipari <notifications@github.com> wrote:

> The original Google Code project seems to indicate the MIT license?
>
> —
>


Yes sorry about that, I noticed the other day that the automatic conversion
from googlecode (more or less forced by google closing down that site:-)
did not preserve the licence information in the projects.

I migrated several projects and they all lost licence info.

Because of its history web-xslt is available under several licences

Use and distribution of this code are permitted under the terms of the <a
href="http://www.w3.org/Consortium/Legal/copyright-software-19980720"
>W3C Software Notice and License</a>.
Or the Apache 2, MIT or MPL 1.1 or MPL 2.0 licences.

This can be seen in the comments at the top of

https://github.com/davidcarlisle/web-xslt/blob/master/ctop/ctop.xsl

But I'll try to get a proper top level licence file organised so you don't
have to look
for comments in random files.

David
